src/api/api.tsx

```diff
@@ -6,8 +9,12 @@ export default class API {
   // Local Python backend
   URL = 'http://127.0.0.1:5000/';
 
-  //Resume
-  addPdfsURL = () => `${this.URL}/upload`;
+  //URLs
+  addPdfURL = () => `${this.URL}/upload`;
+  getApplicantsURL = () => `${this.URL}/applicants`;
+  getApplicantURL = (id: string) => `${this.URL}/applicants/${id}`;
```

Comment:
Routes which handle single objects should be named in singular

Reply:
Fixed it